dataactbroker/handlers/fileGenerationHandler.py

```diff
@@ -123,8 +82,3 @@ def generate_e_file(task, submission_id, job_id, timestamped_name, upload_file_n
         session.commit()
 
         write_csv(timestamped_name, upload_file_name, is_local, fileE.Row._fields, rows)
-
-
-if __name__ in ['__main__', 'jobQueue']:
-    configure_logging()
-    celery_app.conf.update(BROKER_URL=broker_url('localhost'))
```

Comment:
So since this won't be running as its own script anymore, we don't need to call `configure_logging` since it'll be setup prior the calling any function here, which happens in `app.py`.